plugin/evm/validators_manager_test.go

```diff
@@ -0,0 +1,65 @@
+package evm
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/flare-foundation/flare/ids"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func randArray() [20]byte {
+	data := [20]byte{}
+	rand.Read(data[:])
+
+	return data
+}
+
+func randUint64() (uint64, error) {
+	data := make([]byte, 8)
+	_, err := rand.Read(data)
+	return binary.LittleEndian.Uint64(data), err
+}
+
+func TestToSet(t *testing.T) {
+	var (
+		shortIDKeys     [][20]byte
+		testUint64Slice []uint64
+	)
+	for i := 0; i < 3; i++ {
+		tu, err := randUint64()
+		require.NoError(t, err)
+
+		testUint64Slice = append(testUint64Slice, tu)
+		shortIDKeys = append(shortIDKeys, randArray())
+	}
+
+	testValidatorsMap := map[ids.ShortID]uint64{
+		ids.ShortID(shortIDKeys[0]): testUint64Slice[0],
+		ids.ShortID(shortIDKeys[1]): testUint64Slice[1],
+		ids.ShortID(shortIDKeys[2]): testUint64Slice[2],
+	}
+
+	set, err := toSet(testValidatorsMap)
```

Comment:
Should be simplified this way IMO

```suggestion
	var (
		ids []ids.ShortID
		weights []uint64
	)
	for i := 0; i < 3; i++ {
		weight, err := randUint64()
		require.NoError(t, err)
		weights = append(weights, weight)

		id := randID()
		ids = append(ids, id)
	}

	validators := map[ids.ShortID]uint64{
		ids[0]: weights[0],
		ids[1]: weights[1],
		ids[2]: weights[2],
	}

	set, err := toSet(validators)
```

Reply:
This is simplified and moved because this function was moved to vm 
https://github.com/flare-foundation/coreth/pull/21/commits/b5d59c2475c2f081d8ccb493da8daedbd87067ff#diff-7cc8115837d92067c0e48165d5b49fcd63797639f8680ceee734caa4983e9b32R484
Thanks!